lib/transform.js

```diff
@@ -168,3 +169,23 @@ function mapCheckItemIssueToMkdn(issue) {
   }
   return `- [${checked}] #${issue.number}`;
 }
+
+Transform.trelloUsernameToId = function (username) {
+  if (!username) {
+    return;
+  }
+  let ownerInfo = memberlist.find(function (member) {
+    return member.username.includes(username.toLowerCase().slice(1));
+  });
+  return ownerInfo.id;
+};
+
+Transform.trelloIdToUsername = function (id) {
+  if (!id) {
+    return;
+  }
+  let username = memberlist.find(function (member) {
```

Comment:
Here I, I would have done `let member = ` and then `return member.username` because it reads more clearly.

Change if you want, will accept as-is.